challenge.rb

```diff
@@ -6,7 +6,7 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  return age < minimum_age
```

Comment:
Should have been _greater than_ minimum age; however, I won't take points off because the question is a bit ambiguous.